disnake/embeds.py

```diff
@@ -175,6 +187,7 @@ class Embed:
     )
 
     Empty: Final = EmptyEmbed
+    _default_colour: Union[int, Colour] = Empty
```

Comment:
```suggestion
    _default_colour: MaybeEmpty[Colour] = Empty
```
same thing with the other `Union[int, Colour]`, unless I'm missing something